db/indexes.go

```diff
@@ -458,3 +459,15 @@ func replaceSyncTokensQuery(statement string, useXattrs bool) string {
 func replaceIndexTokensQuery(statement string, idx SGIndex, useXattrs bool) string {
 	return strings.Replace(statement, indexToken, idx.fullIndexName(useXattrs), -1)
 }
+
+// Replace $activeOnlyFilter placeholder with activeOnlyFilterFilterExpression if activeOnly is true and the limit
+// is grater than zero. If activeOnly is false or limit is not grater than 0, the $activeOnlyFilter placeholder will
+// be replaced with an empty string in the channel query statement.
+func replaceActiveOnlyFilter(statement string, limit int, activeOnly bool) string {
```

Comment:
I think this function, as well as the activeOnlyFilter const, should be defined in query.go instead of indexes.go.

Reply:
Yeah, make sense. The activeOnlyFilter const has been moved to query.go from indexes.go.